pyirf/benchmarks/angular_resolution.py

```diff
@@ -63,6 +72,8 @@ def angular_resolution(
     # use groupby operations to calculate the percentile in each bin
     by_bin = table[valid].group_by(bin_index[valid])
     for bin_idx, group in zip(by_bin.groups.keys, by_bin.groups):
-        result[key][bin_idx] = np.nanquantile(group["theta"], quantile)
-        result["n_events"][bin_idx] = len(group)
+        for key, value in zip(keys, quantile):
+            result[key][bin_idx] = np.nanquantile(group["theta"], value)
```

Comment:
`np.nanquantile` can compute for multiple quantiles at the same time. Since sorting the input values is the most expensive operation, this is much faster than doing it one-by-one.

This was the main reason to add support for multiple quantiles to the function itself instead of calling it multiple times.